charts/stable/nextcloud/questions.yaml

```diff
@@ -519,6 +508,48 @@ questions:
                   type: int
                   required: true
                   default: 30
+        - variable: opcache
+          label: OPCache Configuration
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              - variable: interned_strings_buffer
+                label: Interned Strings Buffer
+                description: The amount of memory used to store interned strings, in megabytes.
+                schema:
+                  type: int
+                  required: true
+                  default: 32
+              - variable: max_accelerated_files
+                label: Max Accelerated Files
+                description: The maximum number of keys (and therefore scripts) in the OPcache hash table.
+                schema:
+                  type: int
+                  required: true
+                  default: 10000
+              - variable: memory_consumption
+                label: Memory Consumption
+                description: The size of the shared memory storage used by OPcache, in megabytes.
+                schema:
+                  type: int
+                  required: true
+                  default: 10000
```

Comment:
Thank you for the implementation, it's fantastic!
But isn't that default value of 10'000 MB a bit high? Before it was set to  `opcache.memory_consumption=128`
10'000 was probably copied from the vaule above? ( `opcache.max_accelerated_files=10000`)

Reply:
Yep, that’s correct, is a typo.
Can you please either open a PR or an issue so I can remember to fix that tomorrow? THanks